modules/snmp/README.md

```diff
@@ -6,219 +6,254 @@ sidebar_label: "SNMP"
--  use `community` instead of `user`
--  set `options.version` to 1 or 2
+If you need to define many charts using incremental OIDs, you can use the `charts.multiply_range` option.
+
+This is like the SNMPv1/2 [example](#example-using-snmpv12), but the option will multiply the current chart from 1 to 24
+inclusive, producing 24 charts in total for the 24 ports of the switch `192.0.2.1`.
+
+Each of the 24 new charts will have its id (1-24) appended at:
+
+- its chart unique `id`, i.e. `bandwidth_port_1` to `bandwidth_port_24`.
+- its title, i.e. `Switch Bandwidth for port 1` to `Switch Bandwidth for port 24`.
+- its `oid` (for all dimensions), i.e. dimension in will be `1.3.6.1.2.1.2.2.1.10.1` to `1.3.6.1.2.1.2.2.1.10.24`.
+- its `priority` will be incremented for each chart so that the charts will appear on the dashboard in this order.
 
-Example:
 ```yaml
 jobs:
-  - name: local
-    update_every: 3
-    hostname: "10.11.12.8"
+  - name: switch
+    update_every: 10
+    hostname: "192.0.2.1"
+    community: public
     options:
-      port: 1161
       version: 2
-      max_request_size: 60
-    community: "public"
     charts:
-      - title: "example"
-        priority: 1
+      - id: "bandwidth_port"
+        title: "Switch Bandwidth for port"
         units: "kilobits/s"
         type: "area"
-        family: "lan"
-        multiply_range: [1,5]
+        family: "ports"
+        multiply_range: [1, 24]
         dimensions:
           - name: "in"
-            oid: ".1.3.6.1.2.1.2.2.1.10"
+            oid: "1.3.6.1.2.1.2.2.1.10"
             algorithm: "incremental"
             multiplier: 8
             divisor: 1000
           - name: "out"
-            oid: ".1.3.6.1.2.1.2.2.1.16"
+            oid: "1.3.6.1.2.1.2.2.1.16"
             multiplier: -8
             divisor: 1000
 ```
 
-## SNMPv3
-To use SNMPv3:
-
--   use `user` instead of `community`
--   set `options.version` to 3
-
-Example: refer to the first example in this README.
-
-Security levels (`user.level`):
-
--   1 is `noAuthNoPriv`
--   2 is `authNoPriv`
--   3 is `authPriv`
-
-Authentication protocols (`user.auth_proto`):
-
--   "1" is `none`
--   "2" is `md5`
--   "3" is `sha`
--   "4" is `sha224`
--   "5" is `sha256`
--   "6" is `sha384`
--   "7" is `sha512`
+## Data collection speed
 
-Privacy protocols (`user.priv_proto`):
+Keep in mind that many SNMP switches and routers are very slow. They may not be able to report values per second.
+`go.d.plugin` reports the time it took for the SNMP device to respond when executed in the [debug](#troubleshooting)
+mode.
 
--   "1" is `none`
--   "2" is `des`
--   "3" is `aes`
--   "4" is `aes192`
--   "5" is `aes256`
+Also, if many SNMP clients are used on the same SNMP device at the same time, values may be skipped. This is a problem
+of the SNMP device, not this collector. In this case, consider reducing the frequency of data collection (
+increasing `update_every`).
 
-## Testing the configuration
+## Finding OIDs
 
```

Comment:
Should we add a paragraph on why you should need to find an OID or can we assume our users are familiar with this?

Reply:
This is just a copy from https://learn.netdata.cloud/docs/agent/collectors/node.d.plugin/snmp#finding-oids

We assume users have a basic understanding of SNMP protocol.

> why you should need to find an OID

Because it is a crucial part of this plugin configuration. "want to collect" is a hyperlink to the section

<img width="856" alt="Screenshot 2022-03-25 at 10 09 34" src="https://user-images.githubusercontent.com/22274335/160080530-0047f4ee-da2a-4097-8867-7786fbb5c2a1.png">